api/android/api/src/main/jni/nnstreamer-native-api.c

```diff
@@ -30,7 +30,6 @@ GST_PLUGIN_STATIC_DECLARE (amcsrc);
 extern void init_filter_cpp (void);
 extern void init_filter_custom (void);
 extern void init_filter_custom_easy (void);
-extern void init_filter_tflite (void);
```

Comment:
This should be optionally included/excluded by build options along with nn-runtime, snap, and snpe(in the future). The clients want to have multiple versions of .aar depending on the requirements of each application.

Or are you going to make separate .so for each "optional" subplugins in .aar?

Reply:
I added another PR, build option for subplugins.

Reply:
If that's the case, this is good for now.